applications/advanced-concepts/zero-shot-translation.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Building and research
+title: Zero-shot translation
+description: Machine translation without parallel training data
+---
+
+**Zero-shot machine translation** is translation without parallel data between the source and target languages.
+Zero-shot translation is desirable because it can be too costly to create training data for each language pair.
+
+Zero-shot machine translation is an active area of research.
+Zero-shot models are typically lower quality than supervised translation models.
+
+## Approaches
+
+- Bridging: If a language pair `(source, target)` is desired, this approach translates twice: `(source, bridge)`, then `(bridge, target)`. The `bridge` language is also called a pivot language. English is a common choice for the pivot because many languages have training data with English.
+- Multilingual [neural machine translation](/approaches/neural-machine-translation.md) (MNMT): This approach learns a single model for all language pairs. The target language is an input to the model. This approach can produce translations between languages that both have parallel data, but not necessarily parallel data with each other. 
+- Transfer learning: These approaches connect an encoder for the source language to a decoder for the target language. Algorithms are developed to align the latent representations of encoders and decoders across languages.
+
```

Comment:
"Algorithms are developed" is a bit vague.

Reply:
From the quick and dirty lit review it seemed like there were two approaches:
1. Train encoder-decoders for a bunch of parallel texts (many pairs, like MNMT), make an effort to align latent representations, then stitch together the relevant encoder and decoder for the new language pair -- from <div class="csl-entry">Ji, B., Zhang, Z., Duan, X., Zhang, M., Chen, B., &#38; Luo, W. (2020). Cross-Lingual Pre-Training Based Transfer for Zero-Shot Neural  Machine Translation. <i>AAAI</i>. www.aaai.org</div>
2. Train denoising autoencoders on monolingual data and _somehow_ align the latent representations and do the same. I think these are typically called unsupervised MT. A couple refs that I haven't read yet -- https://aclanthology.org/2021.emnlp-main.533/  https://aclanthology.org/D18-1101/  Though also I the general idea

I grouped them together cause they seemed so similar but I didn't see a good naming convention for it and transfer learning seems too generic. I'll see what I can do at least about the wording